session6/src/com/itschool/homeworks6/HomeworkS6Ex2.java

```diff
@@ -0,0 +1,15 @@
+package com.itschool.homeworks6;
+
+public class HomeworkS6Ex2 {
+
+    public static void main(String[] args) {
+
+
+        int firstNumber = 26;
+        int secondNumber = 27;
+        System.out.println(firstNumber == secondNumber);
+        String firstString = "26";
+        String secoundString = "26";
+        System.out.println(firstString.equals(secondNumber));
```

Comment:
I think you wanted to compare here `firstString` with `secoundString`, not with `secondNumber`, because `secoundString` is unused.